Formula/swift.rb

```diff
@@ -338,40 +330,30 @@ def install
           --swift-darwin-module-archs=#{Hardware::CPU.arch}
           --back-deploy-concurrency
           --install-back-deploy-concurrency
-          --lldb-no-debugserver
           --lldb-use-system-debugserver
         ]
         extra_cmake_options += %W[
           -DLLDB_FRAMEWORK_COPY_SWIFT_RESOURCES=0
           -DCMAKE_INSTALL_RPATH=#{loader_path}
         ]
-
-        # Some scripts still reference "python" rather than "python3".
-        # Remove this with Swift 5.7.
-        python_workaround = workspace/"python-workaround"
-        mkdir python_workaround
-        ln_s Utils.safe_popen_read("xcrun", "-find", "python3").chomp, python_workaround/"python"
-        ENV.prepend_path "PATH", python_workaround
       end
       if OS.linux?
         args += %W[
-          --libcxx
+          --libcxx=0
```

Comment:
Unused, I presume?

Reply:
Seems not, or at least not according to https://github.com/apple/swift/commit/e2c5f9bae0e581f8a04658ebf9a1bc116795c26f